src/components/form/form.tsx

```diff
@@ -0,0 +1,124 @@
+import { Button, Field, FormGeneric, FormPanel } from '@mdb/flora';
+import {
+  jobFunctions,
+  institutionTypes,
+  teachingStatuses,
+} from 'src/lib/registration';
+import FormProps from './types';
+import styles from './styles';
+
+interface FieldInterface {
+  name: string;
+  label: string;
+  options?: Array<string>;
+  component: 'text-input' | 'select' | 'text-area' | 'checkbox';
+}
+
+const fields: Array<FieldInterface> = [
+  {
+    name: 'firstName',
+    label: 'First Name',
+    component: 'text-input',
+  },
+  {
+    name: 'lastName',
+    label: 'Last Name',
+    component: 'text-input',
+  },
+  {
+    name: 'email',
+    label: 'Email',
+    component: 'text-input',
+  },
+  {
+    name: 'jobFunction',
+    label: 'Your Job Function',
+    component: 'select',
+    options: jobFunctions,
```

Comment:
So on the current form, when the "Other" option is selected from this list, a new form field appears asking the user to specify. I would confirm with Liz, but at least to have functional parity we would need something like this.

Reply:
@mcolella14 thanks for the feedback. I am going to set up some time with Liz this week to go over everything on Forms, I will bring this up along with your points on field validation.